src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -91,20 +92,20 @@ using OVClassSeveralDevicesTestQueryNetwork = OVClassSeveralDevicesTest;
 using OVClassSeveralDevicesTestDefaultCore = OVClassSeveralDevicesTest;
 
 inline bool supportsAvaliableDevices(ov::runtime::Core& ie, const std::string& deviceName) {
-    auto supportedMetricKeys = ie.get_metric(deviceName, METRIC_KEY(SUPPORTED_METRICS)).as<std::vector<std::string>>();
-    return supportedMetricKeys.end() !=
-           std::find(std::begin(supportedMetricKeys), std::end(supportedMetricKeys), METRIC_KEY(AVAILABLE_DEVICES));
+    auto supported_properties = ie.get_property(deviceName, ov::supported_properties);
+    return supported_properties.end() !=
+           std::find(std::begin(supported_properties), std::end(supported_properties), ov::available_devices);
 }
 
 bool supportsDeviceID(ov::runtime::Core& ie, const std::string& deviceName) {
-    auto supportedConfigKeys =
-            ie.get_metric(deviceName, METRIC_KEY(SUPPORTED_CONFIG_KEYS)).as<std::vector<std::string>>();
-    return supportedConfigKeys.end() !=
-           std::find(std::begin(supportedConfigKeys), std::end(supportedConfigKeys), CONFIG_KEY(DEVICE_ID));
+    auto supported_properties =
+            ie.get_property(deviceName, ov::supported_properties);
+    return supported_properties.end() !=
+           std::find(std::begin(supported_properties), std::end(supported_properties), ov::device::id);
 }
 
 TEST(OVClassBasicTest, smoke_createDefault) {
-    ASSERT_NO_THROW(ov::runtime::Core ie);
+    OV_ASSERT_NO_THROW(ov::runtime::Core ie);
```

Comment:
why do we need custom `OV_ASSERT_NO_THROW` ?

Reply:
`ASSERT_NO_THROW` does not print exception messages.  `OV_ASSERT_NO_THROW` prints